content/en/blog/2023/announcing-notation-v1.0.0.md

```diff
@@ -0,0 +1,95 @@
+
+In addition, `--debug` and `--verbose` flags were added to all CLI commands, providing debug and troubleshooting capability for Notation developers and users respectively.
+
+### Introduced experimental features and switch
+
+Experimental features are intended for testing and evaluation purposes only and should not be used in production environments. Users can enable experimental features in Notation CLI by setting the environment variable `NOTATION_EXPERIMENTAL` to 1. 
+
+There are two major features which are marked as experimental stage but might be useful in some specific scenarios.
+ 
+- Support [signing, listing and verification with OCI image layout](https://notaryproject.dev/docs/how-to/oci-image-layout/) for signing images before they are pushed to a registry. It enables users to sign and verify images in an air-gapped environment.
+- Support [OCI distribution referrers API](https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc2/spec.md#enabling-the-referrers-api), which allows Notation CLI to fetch a list of referrers in an efficient and clean manner.
+
+### Better extensibility and more plugins support
+
+Notation has an [extensible plugin framework design](https://github.com/notaryproject/notaryproject/blob/main/specs/plugin-extensibility.md) which provides plugin interfaces for users, and vendors to implement their own integrations with the solutions they use. Notation has the following plugins available to use now. 
+
+- AWS Signer plugin
+- Azure Key Vault plugin
+- HashiCorp Vault plugin (Experimental)
+
+### Integration with admission controller for Kubernetes usage
+
+As more and more users are requesting to verify and secure image deployment on Kubernetes, the Notary maintainers worked with the [Ratify](https://github.com/deislabs/ratify) and [Kyverno](https://kyverno.io/) team respectively to provide solutions for verifying images signed by Notation before deploying them to Kubernetes. Users will have two different options to build a complete end-to-end image integrity workflow for their environments. See the following two guides for details.
+
+- [Verify CNCF Notary format signatures with Kyverno](https://kyverno.io/docs/writing-policies/verify-images/notary/)
+- [Sign and verify an image with Notation, Ratify, and OPA Gatekeeper](https://ratify-web.netlify.app/blog/mdx-blog-post)
+
+![](https://hackmd.io/_uploads/S1bow5HO2.png)
+
+
+## Security audit report
+
+The Notary project also completed fuzzing audit and security audit in 2023. Some known vulnerabilities had been fixed by the Notary maintainers before Notation v1.0.0. You can find two audit reports as follows.
+
+- [Notary Security Audit Report 2023 (TBD)]()
+- [Notary fuzzing audit report 22-23](https://github.com/notaryproject/notaryproject/tree/main/security/reports/fuzzing/ADA-fuzzing-audit-22-23.pdf)
+
+## Community status
+
+Add CNCF annual review report (PR link)
```

Comment:
We can remove this since it is not ready to avoid confusion

Reply:
I think the PR for CNCF annual review report is also critical as CNCF TOC has an issue created for this one annual review. I will add the PR link soon

Reply:
Removed this section to keep limited scope of the release blog post